src/Integration/Binding/BindCommand.cs

```diff
@@ -1,159 +0,0 @@
-//-----------------------------------------------------------------------
-// <copyright file="BindCommand.cs" company="SonarSource SA and Microsoft Corporation">
-//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
-//   Licensed under the MIT License. See License.txt in the project root for license information.
-// </copyright>
-//-----------------------------------------------------------------------
-
-using Microsoft.TeamFoundation.Client.CommandTarget;
-using SonarLint.VisualStudio.Integration.Progress;
-using SonarLint.VisualStudio.Integration.Resources;
-using SonarLint.VisualStudio.Integration.Service;
-using SonarLint.VisualStudio.Integration.TeamExplorer;
-using SonarLint.VisualStudio.Integration.WPF;
-using SonarLint.VisualStudio.Progress.Controller;
-using System;
-using System.Diagnostics;
-using System.Linq;
-
-namespace SonarLint.VisualStudio.Integration.Binding
-{
-    internal class BindCommand : HostedCommandBase, IBindingWorkflowExecutor
```

Comment:
The BindCommand/ConnectCommand were renamed to BindController/ConnectionController.